### PR TITLE
Shop owner login and registration

### PR DESCRIPTION
**Pull request by HKanwal**
See #38

### CODE REVIEW

**Review by jere-mie — APPROVED** [review]
LGTM 🚀🚀🚀

**Comment by HKanwal** [comment]
@jere-mie Oh no, this is embarrassing. My sleep deprived commit messages have been exposed.

File: src/next-app/components/EmailsField.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  onBlur?: () => void;
+
+  /** Makes the first input highlighted red */
+  error?: boolean;
+
+  /** Adds a red asterisk */
+  required?: boolean;
+};
+
+const EmailsField = (props: EmailsFieldProps) => {
```
Comment (tevisdoe) [inline src/next-app/components/EmailsField.tsx:23]: You should probably update this emails field component to make use of the email validation utility you made.

**Review by tevisdoe — APPROVED** [review]
I did give this decent enough look over, but... it's a ton of code. The actual application looks nice, though. I only left 1 comment, and honestly, it's a nitpick for a POC component, so I'm leaving it up to you if you actually want to make the change or not.

File: src/next-app/components/EmailsField.tsx (hunk 1 above)
Comment (HKanwal) [reply src/next-app/components/EmailsField.tsx:23]: I'll just merge it for now for the sake of getting it into main so everyone can work on their parts for the POC.

**Pull request merged**